easybuild/easyconfigs/t/TensorRT/TensorRT-4.0.1.6-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,36 @@
+name = 'TensorRT'
+version = '4.0.1.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://developer.nvidia.com/tensorrt'
+description = """NVIDIA TensorRT is a platform for high-performance deep learning inference"""
+
+sources = ['%(name)s-%(version)s.Ubuntu-16.04.4.x86_64-gnu.cuda-9.2.cudnn7.1.tar.gz']
+checksums = ['fc5fd2ba86225ed62e60870cc24896107770a59829363fcb49229c82bb1f4fb5']
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
```

Comment:
@akesandgren This should move up above `sources` to adher to our common easyconfigs "style"